app/root.tsx

```diff
@@ -133,7 +133,7 @@ function Head() {
   return (
     <head>
       <meta charSet="utf-8" />
-      <meta name="viewport" content="width=device-width,initial-scale=1" />
+      <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
```

Comment:
huh, why do we need to disable zoom-out on mobile? there is no issue linked in the commit...